Sources/Rules/SpacingGuards.swift

```diff
@@ -0,0 +1,33 @@
+// Created by @NikeKov on 01.08.2024
+// Copyright © 2024 Nick Lockwood. All rights reserved.
+
+import Foundation
+
+public extension FormatRule {
+    static let spacingGuards = FormatRule(help: "Remove space between guard and add spaces after last guard.",
```

Comment:
I think "guardSpacing" would be a slightly better name for the rule

```suggestion
    static let guardSpacing = FormatRule(help: "Remove space between guard and add spaces after last guard.",
```

Reply:
Hope this is not mandatory thread. I wanna leave how it now)